### DISCUSSION TITLE
Restrict editing specific fields?

### DISCUSSION THREAD

**Discussion by llamafilm** [netbox-community/netbox#17188] [Q&A]
Is it possible to make specific fields read-only?  For context: I'm using the `Facility` field on each Location to reference the unique ID from our real estate system, and it's filled in Netbox by automation.  So I'd like to prevent users from changing it.

**Comment by candlerb** [comment]
You can do it with [custom fields](https://netboxlabs.com/docs/netbox/en/stable/customization/custom-fields/#visibility-editing), but I don't think it's possible with core attributes. And I don't think you can use Validators to prevent changes either, as they don't get the before-and-after view.
